src/utils/languageMismatch.ts

```diff
@@ -0,0 +1,74 @@
+import { commands, ConfigurationTarget, ExtensionContext, languages, TextDocument, window, workspace } from "vscode";
+import { ProjectLabelInfo } from "../definitions/ProjectLabelInfo";
+
+export abstract class LanguageMismatch {
+  static options: LanguageMismatchOption;
+
+  static setLanguageMismatch(configName: string, value: LanguageMismatchOption): Thenable<void> {
+    return this.saveToConfig(configName, value);
+  }
+
+  static saveToConfig<T>(configName: string, value: T): Thenable<void> {
+    return workspace.getConfiguration().update(configName, value, ConfigurationTarget.Global);
+  }
+}
+
+enum LanguageMismatchOption {
+  ignore = "ignore",
+  force = "force",
```

Comment:
Renamed option to `force` for consistency in both Qute and Quarkus language mismatch